tests/test_var.py

```diff
@@ -958,6 +958,21 @@ def test_all_number_operations():
     )
 
 
+@pytest.mark.parametrize(
+    "var,expected",
```

Comment:
```suggestion
    ("var", "expected"),
```

although csv string does work, the preferred way i think is to use a tuple of args